packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx

```diff
@@ -387,7 +387,7 @@ class UserProfileComponent extends React.Component<Props, State> {
                       </InformationValue>
                     </InformationHolder>
                     <InformationHolder>
-                      <InformationTitle paddingRight={113}>
+                      <InformationTitle paddingRight={208}>
```

Comment:
Why do we need this change ? Can you upload a screen shot for this ?

Reply:
After padding: 
![ocrvs-2169](https://user-images.githubusercontent.com/46846503/125403807-c2c38780-e3d7-11eb-856a-e594b3c2e05a.png)